modules/javafx.controls/src/main/java/javafx/scene/control/TreeTablePosition.java

```diff
@@ -77,6 +77,17 @@ public TreeTablePosition(@NamedArg("treeTableView") TreeTableView<S> treeTableVi
         nonFixedColumnIndex = treeTableView == null || tableColumn == null ? -1 : treeTableView.getVisibleLeafIndex(tableColumn);
     }
 
+    // A call to 'treeTableView.getTreeItem(row)' may result in TreeModificationEvent being triggered.
+    // This causes issue by triggering a new TreeModificationEvent while one TreeModificationEvent
+    // is being handled currently.
```

Comment:
The first three lines of added comments don't really belong here. I would just document the new constructor as a copy-like constructor (with a different row) and mention that it is used by the TreeTableView::sort method.

Reply:
Updated the doc in next commit, please take a look